test/dynamo/test_replay_record.py

```diff
@@ -47,6 +47,9 @@ def tearDownClass(cls):
         cls._exit_stack.close()
 
     def check_replay(self, fn, *args, exp_exc_name=None):
+        raise unittest.SkipTest(
+            "Errors used in this class have been fixed, todo: re-enable"
```

Comment:
cc @mlazos TODO

Reply:
How have these errors been fixed? One is a bona fide dimension mismatch, and the other is using tensors as dict keys which I don't think is supported as well. (We should only support parameters as keys)

Reply:
Hmm I think I was on a bad local base, this no longer needs to be skipped.